crates/vm/levm/src/opcode_handlers/stack_memory_storage_flow.rs

```diff
@@ -207,6 +205,34 @@ impl VM {
 
         self.increase_consumed_gas(current_call_frame, base_dynamic_gas)?;
 
+        // Gas Refunds
+        let mut gas_refunds = U256::zero();
+        if value != storage_slot.current_value {
+            if storage_slot.current_value == storage_slot.original_value {
+                if storage_slot.original_value != U256::zero() && value == U256::zero() {
+                    gas_refunds += U256::from(4800);
+                }
+            } else if storage_slot.original_value != U256::zero() {
+                if storage_slot.current_value == U256::zero() {
+                    gas_refunds -= U256::from(4800);
+                } else if value == U256::zero() {
+                    gas_refunds += U256::from(4800);
+                }
+            } else if value == storage_slot.original_value {
+                if storage_slot.original_value == U256::zero() {
+                    gas_refunds += U256::from(20000) - U256::from(100);
+                } else {
+                    gas_refunds += U256::from(5000) - U256::from(2100) - U256::from(100);
+                }
+            }
+        };
```

Comment:
Nit. `U256` has a `is_zero` method.
```suggestion
        if value != storage_slot.current_value {
            if storage_slot.current_value == storage_slot.original_value {
                if !storage_slot.original_value.is_zero() && value.is_zero() {
                    gas_refunds += U256::from(4800);
                }
            } else if !storage_slot.original_value.is_zero() {
                if storage_slot.current_value.is_zero() {
                    gas_refunds -= U256::from(4800);
                } else if value.is_zero() {
                    gas_refunds += U256::from(4800);
                }
            } else if value == storage_slot.original_value {
                if storage_slot.original_value.is_zero() {
                    gas_refunds += U256::from(20000) - U256::from(100);
                } else {
                    gas_refunds += U256::from(5000) - U256::from(2100) - U256::from(100);
                }
            }
        };
```

Reply:
https://github.com/lambdaclass/lambda_ethereum_rust/pull/1113/commits/51dd2c1f4fc9fbd8a63b00373da5facb206ba896